src/components/Task.jsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable no-magic-numbers */
+import {
+  Card, Row, Col, Input, Select,
+} from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+
+import TaskDisplay from './TaskDisplay';
+
+const { Option } = Select;
+
+function Task(props) {
+  const {
+    handleMinutesChange,
+    handleOnDelete,
+    handleTaskTitleChange,
+    minutesChange,
+    task,
+    taskList,
+    titleChange,
+    totalScheduleTime,
+    totalTasksTime,
+  } = props;
+
+  const timeValues = [25, 30, 35, 45, 50, 60, 75, 90];
+
+  const timeOptionsDisplay = timeValues.map((value) => {
+    if (totalScheduleTime > value) {
+      if (totalScheduleTime - totalTasksTime > value) {
+        return <Option value={value}>{value}</Option>;
+      }
+    }
+
+    return null;
+  });
+
+  if (task.createdAt) {
+    return <TaskDisplay handleOnDelete={handleOnDelete} task={task} taskList={taskList} />;
+  }
+
+  return (
+    <ParentContainer>
+      <CardContainer>
+        <RowContainer>
+          <Col span={10}>
+            <Input
+              placeholder="Task Title"
+              value={titleChange}
+              onChange={handleTaskTitleChange}
+            />
+          </Col>
+          <SelectMinutesColumn span={12}>
+            <Select
+              defaultValue="25"
+              value={minutesChange}
+              onChange={handleMinutesChange}
+            >
+              {timeOptionsDisplay}
+            </Select>
+            <MinutesLabel>minutes</MinutesLabel>
+          </SelectMinutesColumn>
+        </RowContainer>
+      </CardContainer>
+    </ParentContainer>
+  );
+}
+
+const ParentContainer = styled.div`
+  padding-bottom: 0.5em;
+`;
+
+const CardContainer = styled(Card)`
```

Comment:
In situations like this, our team prefix `style`. This is just a preference, not a best practice. For example, 

`const styledCard = styled(Card)`
`const styledRow = styled(Row)`

Reply:
Agreed Mart but we should also be consistent with our casing since these are components. should be StyledCard and StyledRow

Reply:
Nice catch, Leigh. Yep, they should be in pascal case